firmware/mw_dual_motor_torque_ctrl/src/dual_motor_torque_ctrl.c

```diff
@@ -568,6 +568,29 @@ void main(void)
 	// At the beginning, there are no errors
 	gErrors.all = 0;
 
+	// Enable the motors programmatically.
+	gMotorVars[HAL_MTR1].Flag_enableSys = true;  // enable the system in general
+	gMotorVars[HAL_MTR1].Flag_Run_Identify = true;  // enable the motor
+	gMotorVars[HAL_MTR2].Flag_enableSys = true;  // enable the system in general
+	gMotorVars[HAL_MTR2].Flag_Run_Identify = true;  // enable the motor
+
+	// Speedup the initialization time by a factor of 10.
+	gUserParams[HAL_MTR1].ctrlWaitTime[CTRL_State_OffLine] /= 10;
+	gUserParams[HAL_MTR2].ctrlWaitTime[CTRL_State_OffLine] /= 10;
+
+	// Lower the current used for alignment to be less agressive and demand less
+	// power from the power supply.
+	gUserParams[HAL_MTR1].maxCurrent_resEst /= 5.;
+	gUserParams[HAL_MTR2].maxCurrent_resEst /= 5.;
```

Comment:
Depending on the robot, this actually needs to be aggressive as it needs to be stronger than the gravitational pull on the joints (this is, for example, relevant on the TriFingerPro robots, where the links are rather heavy).

Reply:
Apart from this, I think it would be better to change the original value in the motor configuration instead of modifying it here.